crates/pallet-domains/src/lib.rs

```diff
@@ -1425,6 +1463,20 @@ mod pallet {
                         }
                     }
 
+                    if let Err(e) = charge_bundle_storage_fee::<T>(
+                        opaque_bundle.operator_id(),
+                        Zero::zero(),
+                        // TODO: use `opaque_bundle.size()` when deposit to storage fund works
+                    ) {
+                        log::debug!(
+                            target: "runtime::domains",
+                            "Operator {} unable to pay for the bundle storage fee, domain id {:?}, error: {e:?}",
```

Comment:
It would be nice to have the operator only include as many tx into the bundle as he can pay for. Not a blocker, this current impl is in line with the spec.

Reply:
Leave a TODO for now as this requires getting the `transaction_byte_fee` from the consensus chain and is duplicated with #2384